tests/test_unit_conversion.py

```diff
@@ -1,80 +1,1235 @@
+            assert str(silly_names.equations[0]) == 'Eq(Derivative(_env_ode$sv1, _environment$time), _1.0)'
+            state_symbols = silly_names.get_state_symbols()
+            assert len(state_symbols) == 1
+            assert symbol_a in state_symbols
+            assert silly_names.get_free_variable_symbol() == symbol_t
+            return True
+
+        volt_unit = silly_names.get_units('volt')
+        original_var = silly_names.get_symbol_by_name('env_ode$sv1')
+
+        assert test_original_state(silly_names)
+        # change mV to V
+        silly_names.convert_variable(original_var, volt_unit, DataDirectionFlow.INPUT)
+        assert len(silly_names.variables()) == 7
+        symbol_a = silly_names.get_symbol_by_cmeta_id('sv11')
+        assert silly_names.get_initial_value(symbol_a) == 0.002
+        assert symbol_a.units == 'volt'
+        assert symbol_a.name == 'env_ode$sv1_converted_a'
+        symbol_t = silly_names.get_symbol_by_cmeta_id('time')
+        assert symbol_t.units == 'ms'
+        assert symbol_t.name == 'environment$time'
+        symbol_orig = silly_names.get_symbol_by_name('env_ode$sv1')
+        assert symbol_orig.units == 'mV'
+        assert silly_names.get_symbol_by_name('env_ode$sv1_converted')
+        assert silly_names.get_symbol_by_name('env_ode$sv1_orig_deriv')
+        symbol_derv = silly_names.get_symbol_by_name('env_ode$sv1_orig_deriv_a')
+        assert symbol_derv.units == 'mV / ms'
+        assert not silly_names.get_initial_value(symbol_derv)
+        assert len(silly_names.equations) == 3
+        assert str(silly_names.equations[0]) == 'Eq(_env_ode$sv1, 1000.0*_env_ode$sv1_converted_a)'
+        assert str(silly_names.equations[1]) == 'Eq(_env_ode$sv1_orig_deriv_a, _1.0)'
+        assert str(silly_names.equations[2]) == 'Eq(Derivative(_env_ode$sv1_converted_a, _environment$time), ' \
+                                                '0.001*_env_ode$sv1_orig_deriv_a)'
+
+    # def test_missing_units(self, model_missing_units, literals_model):
+    #     """ Tests the Model.add_output function that changes units.
+    #     In this case the model needs to add the unit it wants to change to.
+    #     e.g.
+    #         var time: ms {pub: in};
+    #         var{sv11} sv1: mV {init: 2};
+    #         var{current} x: pA;
+    #         var y: per_pA;
+    #
+    #         ode(sv1, time) = 1{mV_per_ms};
+    #         x = 1{pA};
+    #         y = 1{dimensionless}/x;
+    #
+    #     change x from pA to nA
+    #         var time: ms {pub: in};
+    #         var{sv11} sv1: mV {init: 2};
+    #         var x: pA;
+    #         var y: per_pA;
+    #         var{current} x_converted: nA
+    #
+    #         ode(sv1, time) = 1 :mV_per_ms;
+    #         x = 1000 * x_converted: pA;
+    #         y = 1{dimensionless}/x;
+    #         x_converted = 0.001 * 1 : nA
+    #     """
+    #     pA_unit = model_missing_units.get_units('pA')
+    #     nA_unit = literals_model.get_units('nA')
```

Comment:
So if we want to test this, I'd instead do something like
```
nA_unit = model_missing_units.units.ureg('nanoampere')
```
and not use the `literals_model` fixture.

If we get rid of the test, then remember to remove the fixture & CellML file.